### PR TITLE
Merges all conflicts

### PR DESCRIPTION
**Pull request by hlhartley**

### CODE REVIEW

File: src/Header.js [hunk 1]
```diff
@@ -7,23 +7,6 @@ import SearchByParlor from './SearchByParlor';
 class Header extends Component {
   constructor() {
     super();
```
Comment (ericweissman) [inline src/Header.js]: @hlhartley was removing all of this code intentional?

File: src/css/Card.scss [hunk 2]
```diff
@@ -18,37 +18,6 @@
     &:hover {
         opacity: 1;
     }
-<<<<<<< HEAD
```
Comment (ericweissman) [inline src/css/Card.scss]: @hlhartley same here - do we want to remove all of this code?

File: src/css/Header.scss [hunk 3]
```diff
@@ -1,24 +1,13 @@
 @import url('https://fonts.googleapis.com/css?family=Lobster');
 @import '_variables.scss';
 
-<<<<<<< HEAD
-.header {
```
Comment (ericweissman) [inline src/css/Header.scss]: @hlhartley here as well. I think we really just want to remove line 4 and line 13?

File: src/css/ParlorCard.scss [hunk 4]
```diff
@@ -12,25 +12,6 @@
 
 .parlor-card {
   align-items: center;
-<<<<<<< HEAD
```
Comment (ericweissman) [inline src/css/ParlorCard.scss]: @hlhartley do we want to remove just lines 15 and 33?

File: src/css/ParlorDetail.scss [hunk 5]
```diff
@@ -5,13 +5,6 @@
   background-color: $light-yellow;
   border: 2px solid black;
   bottom: 0;
-<<<<<<< HEAD
-  background-color: #fff8de;
```
Comment (ericweissman) [inline src/css/ParlorDetail.scss]: @hlhartley just lines 8 and 14?

File: src/css/ParlorDetail.scss [hunk 6]
```diff
@@ -24,25 +17,15 @@
   width: 90%;
   z-index: 3;
    a {
-<<<<<<< HEAD
```
Comment (ericweissman) [inline src/css/ParlorDetail.scss]: @hlhartley just 27 and 30? Same for below:

30?
32?
39?
42?
45?
56?
61?
66?

File: src/Header.js (hunk 1 above)
Comment (hlhartley) [reply src/Header.js]: I think I figured it out.
In one of the previous pull requests the merge conflicts were unresolved before merging to master. That's why we were having duplicates.
I resolved the conflicts in this separate branch, deleting the duplicates, which is why it shows that code being deleted. However, the code we still want to keep (ex. in Header.js is still there - you can check the branch file and go into Header.js before merging this).
If you merge this branch, we should have no more conflicts.

Comment (ericweissman) [reply src/Header.js]: Sounds good! I will merge and we can clean up anything unexpected that we overlooked.

**Pull request merged**